packages/jest-preset-mc-app/setup-tests.js

```diff
@@ -1,4 +1,5 @@
 const util = require('util');
+const { MessageChannel } = require('worker_threads');
```

Comment:
This is needed to emulate how we create a private messages channel between host and Custom View in the test environment.